pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -79,6 +79,7 @@ in
 , enableSeparateBinOutput ? false
 , enableSeparateDataOutput ? false
 , enableSeparateDocOutput ? doHaddock
+, enableSeparateDistOutput ? false
```

Comment:
I guess this is slightly different  than options like `enableSeparateBinOutput`, `enableSeparateDataOutput`, etc.

Even if  `enableSeparateBinOutput` and `enableSeparateDataOutput` are set to `false`, the binary and data files still get into the single, main output.  But if `enableSeparateDistOutput` is `false`, then the `dist/build/` directory just completely doesn't get output at all.

I think this is fine as-is (and is likely what most users would expect to happen), but maybe you could add a comment here just clarifying this?

Reply:
I think it should be renamed. The other ones cause something that would be installed anyways to go to a separate output. This flag moves something that would not have been "installed" otherwise to be moved into an output.

Maybe something starting with `do`? Although there ought to be something less awkward and ambiguous than `doInstallDist`…

Reply:
If we were to go the `doInstallDist` route, then for consistency we would want to add both `doInstallDist` *and* `enableSeparateDistOutput`.  I can make that change, but I just wanted to check if that would be alright with you two